packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -401,9 +398,7 @@ export const downloadCsv = async (
             status: SchedulerJobStatus.ERROR,
             details: { createdByUserUuid: payload.userUuid, error: e },
         });
-
-        // do not throw error to avoid retrying
```

Comment:
@rephus why don't we want to retry this job?

Reply:
because we need to make sure our polling on the frontend is compatible with this job retrying (it should if we only check for completed + url)  . I'm just not 100%  this will work, so I disabled it for now

Reply:
Since we create that job with `maxAttempts: 1,` it will be fine since there won't be a retry.